### DISCUSSION TITLE
how to remove github account access frome unwanted website

### DISCUSSION THREAD

**Discussion by Chavariya-Daxesh** [community/community#140040] [Other Feature Feedback, Questions, & Ideas]
i want to remove access for unwanted websites in GitHub account, how to remove

**Comment by yamadatta** [comment]
**-** Click your profile picture (top-right corner)
**-** Select "Settings" from the dropdown.
**-** In the left sidebar, click on "Applications"
**-** Under "Authorized OAuth Apps", find the app you want to remove
**-** Click the app, then click "Revoke access"

Hopefully this will help!

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Chavariya-Daxesh ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
